libtbb.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: libtbb
+  version: 2021.7.0
+  epoch: 0
+  description: "oneAPI Threading Building Blocks"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+      - "*"
+      attestation: TODO
+      license: GPL-2.0-only
+  dependencies:
+    runtime:
+  
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - cmake
+      - samurai
+  
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/oneapi-src/oneTBB/archive/v${{package.version}}.tar.gz
+      expected-sha256: 2cae2a80cda7d45dc7c072e4295c675fff5ad8316691f26f40539f7e7e54c0cc
+  - name: 'Set version and types'
+    runs: |
+      sed -e "s@#define MALLOC_UNIXLIKE_OVERLOAD_ENABLED __linux__@@"  -i src/tbbmalloc_proxy/proxy.h
+  - name: 'Configure libtbb'
+    runs: |
+         CFLAGS="$CFLAGS -flto=auto" \
+         CXXFLAGS="$CXXFLAGS -flto=auto" \
+         cmake -B build -G Ninja \
+          -DCMAKE_C_COMPILER=gcc \
+          -DCMAKE_CXX_COMPILER=g++ \
+          -DCMAKE_BUILD_TYPE=MinSizeRel \
+          -DCMAKE_INSTALL_PREFIX=/usr \
+          -DCMAKE_INSTALL_LIBDIR=lib \
+          -DTBB4PY_BUILD=OFF \
+          -DTBB_STRICT=OFF \
+          -DTBB_TEST=OFF
+  - runs: |
+      cmake --build build
+  - runs: |
+      DESTDIR="${{targets.destdir}}" cmake --install build
+
+subpackages:
+  - name: "libtbb-dev"
+    description: "headers for mimalloc2"
```

Comment:
header for libtbb?

Reply:
My bad, i was copying bits n bobs and forgot about that

Reply:
Fixed it